README.md

```diff
@@ -7,7 +7,7 @@ Provides useful Halogen Hooks and other utilities commonly used in hooks (e.g. a
 - `useDebouncer` - run an action only when a specified amount of time has passed without receiving any new updats/values (e.g. search the database using user's query after 300 ms of no input).
 - `useThrottle` - run an action with the next value after ignoring new values for a specified amount of time (e.g. handle 1 scroll event every 300 ms).
 - `useEvent` - run a callback whenever a new value is pushed (e.g. allow a library to publish events within it to end-users of the library).
-- `useStateFn` / `useModifyState` / `usePutState` - allows `state /\ modifyState <- useState` behavior that was implemented in `halogen-hook` `v0.3.0` and then reverted in `v0.4.0`.
+- Convenience function `useStateFn` and shorthand `useModifyState_` / `useModifyState` / `usePutState`. These provide a `ModadState` update function rather than a stateID. This is similar to the `react-basic-hooks` API, where `useState` in that framework behaves like `useModifyState_` here. Use these functions if you only need a single way to update state and prefer more concise code.
```

Comment:
```suggestion
- Convenience function `useStateFn` and shorthand `useModifyState_` / `useModifyState` / `usePutState`. These provide a `MonadState` update function rather than a stateID. This is similar to the `react-basic-hooks` API, where `useState` in that framework behaves like `useModifyState_` here. Use these functions if you only need a single way to update state and prefer more concise code.
```